Description: fix 'make test' failure due to expired certificate cert0.pem
Author: Antonio Borneo <borneo.antonio@gmail.com>
Origin: http://lists.unix-ag.uni-kl.de/pipermail/vpnc-devel/2011-September/003609.html

--- a/Makefile
+++ b/Makefile
@@ -117,8 +117,8 @@
 	rm -rf vpnc-$*
 
 test : all
-	./test-crypto test/cert.pem test/cert0.pem test/cert1.pem test/cert2.pem test/root.pem
+	./test-crypto test/cert.pem test/cert1.pem test/cert2.pem test/root.pem
 	#./test-crypto test/cert.pem test/cert0.crt test/cert1.crt test/cert2.crt test/root.crt
 
 dist : VERSION vpnc.8 vpnc-$(RELEASE_VERSION).tar.gz
 
